.github/workflows/flower.yml

```
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        python: [3.7.15, 3.8.15, 3.9.16, 3.10.9]
+        python: [3.7.17, 3.8.15, 3.9.16, 3.10.9]
```

Are there also new versions for 3.9 and 3.10?
```suggestion
        python: [3.7.17, 3.8.17, 3.9.16, 3.10.9]
```